ext/opcache/tests/func_info.phpt

```diff
@@ -0,0 +1,85 @@
+--TEST--
+Test that return types in zend_func_info.c match return types in stubs
+--INI--
+opcache.enable=1
+opcache.enable_cli=1
+opcache.optimization_level=-1
+--SKIPIF--
+<?php require_once('skipif.inc'); ?>
+--FILE--
+<?php
+
+$extensions = [
```

Comment:
While not loop over all get_defined_functions?

Reply:
The reason why I chose reflection is because I wanted to avoid calling functions that are surely not defined in zend_func_info. But `get_defined_functions()` will be a good choice because it will also test the `zpp / arginfo mismatch` issues.

Reply:
I'll add a different test for  checking `zpp / arginfo mismatch` where I'lll call the defined functions.